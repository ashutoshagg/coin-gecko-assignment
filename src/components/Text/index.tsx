import React from "react";

interface IProps {
  label: string | undefined;
  value: string | undefined;
}

const Text = ({ label = "", value = "" }: IProps) => {
  return (
    <div>
      <strong>{label}</strong> {value}
    </div>
  );
};

export default Text;
