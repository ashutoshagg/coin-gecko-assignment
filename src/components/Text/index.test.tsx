import React from "react";
import renderer from "react-test-renderer";
import Text from "./index";

describe("Components/Text", () => {
  it("should match snapshot", () => {
    const component = renderer.create(<Text label="Coin" value="value" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
