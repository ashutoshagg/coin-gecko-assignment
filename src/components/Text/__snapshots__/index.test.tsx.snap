// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Text should match snapshot 1`] = `
<div>
  <strong>
    Coin
  </strong>
   
  value
</div>
`;

exports[`Text should match snapshot 1`] = `
<div>
  <strong>
    Coin
  </strong>
   
  value
</div>
`;
